src/Domain/MinimumGridSpacing.hpp

```diff
@@ -28,18 +29,27 @@ double minimum_grid_spacing(
 
 namespace domain {
 namespace Tags {
+// @{
 /// \ingroup ComputationalDomainGroup
 /// \ingroup DataBoxTagsGroup
 /// The minimum coordinate distance between grid points.
 template <size_t Dim, typename Frame>
-struct MinimumGridSpacing : db::ComputeTag {
-  static std::string name() noexcept { return "MinimumGridSpacing"; }
-  static auto function(
-      const ::Mesh<Dim>& mesh,
+struct MinimumGridSpacing : db::SimpleTag {
+  using type = double;
+};
+
+template <size_t Dim, typename Frame>
+struct MinimumGridSpacingCompute : MinimumGridSpacing<Dim, Frame>,
+                                   db::ComputeTag {
+  using base = MinimumGridSpacing<Dim, Frame>;
+  using return_type = double;
```

Comment:
Is the `return_type` alias still needed? If I remember correctly, you are (planning to?) remove it and just have the `type` alias, correct?

Reply:
yes, will be removed soon